scripts/prepare_sector_network.py

```diff
@@ -2048,9 +1876,7 @@ def add_services(n, costs):
     )
     co2 = (
         p_set_biomass.sum().sum() * costs.at["solid biomass", "CO2 intensity"]
-    ) / 8760  # *n.snapshot_weightings.objective[0] #TODO change the way pset is sampled here

```

Comment:
We often had the "TODO: change the way pset is sampled here". Is this obsolete or should we still be careful?